src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -1,89 +1,101 @@
 /* eslint-disable class-methods-use-this */
 /* eslint-disable no-unused-vars */
 // import { debounce } from 'lodash';
-import LikeButton from '../../sdk/button.iframe';
-import eventCenter from '../event-center/index';
+import { renderYouTubeButton } from './render';
 
 class YoutubePlugin {
   youtubeStyle!: HTMLElement;
 
-  failCounter!: number;
-
   insertStyle() {
     this.youtubeStyle = document.createElement('style');
     this.youtubeStyle.innerHTML = `
               #meta-contents {
                 display: flex;
                 flex-direction: row;
                 justify-content: space-between;
+                padding-top:40px;
               }
-              .button-container {
+              .button-container{
                 display: flex;
                 flex-direction: column;
                 justify-content: flex-end;
                 align-items: flex-end;
-                height: 100%;
-                width: 47%;
+                width: 605px;
                 margin-left: 20px;
-                text-align:left;
+                text-align: left;
                 border-left: 1px solid var(--yt-spec-10-percent-layer);
                 padding-left: 14px;
+                background: #eaf4f6;
+                border: 1px solid #eee;
+                padding: 11px;
+                box-shadow: rgb(0 0 0 / 24%) 0px 3px 8px;
+                border-radius: 22px;
+                box-sizing: content-box;
               }
-              .liker-tips {
-                text-align: left;
-                z-index: 10;
-              }
-              .liker-tips-title {
+              li:hover{
                 cursor: pointer;
-                -webkit-transition: opacity,color .2s ease-in-out;
-                transition: opacity,color .2s ease-in-out;
-                color: #28646e;
-                font-size: 16px;
-                line-height: 1.5em;
-                font-weight: 600;
+                text-decoration: underline;
               }
```

Comment:
What is this style for? Seem too general

Reply:
For tips component but I could make it specific